lib/utilities/inspec_util.rb

```diff
@@ -192,59 +191,69 @@ def self.control_finding_details(control, control_clk_status)
     # @todo Allow for the user to pass in a hash for the desired mapping of text
     # values to numbers or to override our hard coded values.
     #
-    def self.get_impact(severity)
-      return float_to_impact(severity) if severity.is_a?(Float)
+    def self.get_impact(severity, use_cvss_terms = true)
+      return float_to_impact(severity, use_cvss_terms) if severity.is_a?(Float)
 
-      return string_to_impact(severity) if severity.is_a?(String)
+      return string_to_impact(severity, use_cvss_terms) if severity.is_a?(String)
```

Comment:
I really feel like all of these being keyword instead of positional arguments would make the code a lot easier to read.